projects/node-http-exercise/node-http-exercise.md

```diff
@@ -0,0 +1,255 @@
+# Node HTTP exercise
+
+## Primary Goal
+
+- To build **an API with a given set of endpoints**.
+- Working with APIs is the daily bread of any full stack developer. Although the Express module is highly used in the industry and makes things much easier, to learn how to build one from scratch will pay you back in spades, and can be really useful to you further down the road.
```

Comment:
Personally, I'd prefer to remove the term `API` from this lesson entirely as it was really meant to learn about HTTP servers, and the fact that some of the endpoints, like time.json, are API-like is not as important here. @alodahl I think you mentioned "API" in one of your comments. Is that okay to just keep that out of scope? I forget the order of the lessons.

Reply:
fixed in [799330c](https://github.com/Techtonica/curriculum/pull/1406/commits/799330cf2b82a0e0d74d3cd2e258f97adc3e70db)